lldb/test/API/python_api/sbstructureddata/TestStructuredDataAPI.py

```diff
@@ -110,6 +110,23 @@ class MyRandomClass:
         self.assertTrue(my_random_class)
         self.assertEqual(my_random_class.payload, MyRandomClass.payload)
 
+        example_arr = [1, 2.3, "4", {"5": False}]
+        arr_str = json.dumps(example_arr)
+        s.Clear()
+        s.Print(arr_str)
+        example = lldb.SBStructuredData()
+
+        # Check SetFromJSON API for dictionaries, integers, floating point
+        # values, strings and arrays
+        error = example.SetFromJSON(s)
+        if not error.Success():
+            self.fail("FAILED:   " + error.GetCString())
```

Comment:
```suggestion
        self.assertSuccess(example.SetFromJSON(s))
```

Reply:
I see you've resolved this comment, but I don't actually understand how. Is your code in some way better than assertSuccess (which exists precisely so it can print the error message when the SBError is in failure state)?

Reply:
Ah, maybe I see... You've changed one instance to assertSucess, but then added another one with the old pattern, right?

If so, then just change the other one as well, and this is good to go :)